CHANGES.md

```diff
@@ -258,16 +260,15 @@ Directory Structure Changes
 
 Cray-specific Changes
 ---------------------
-* 'ugni' no longer requires any heap memory to be registered
-  (see http://chapel.cray.com/docs/1.13/platforms/cray.html)
 * streamlined 'ugni' resource management, dramatically improving performance
-* hugepages is now optional with the 'ugni' communication layer
+* hugepages are now optional with the 'ugni' communication layer
```

Comment:
Okay, yeah, this is probably the better way to go: to talk about the user-facing aspect of the change, which comes down to that they don't need a hugepage module loaded.  And I think ultimately this is less important than the 'ugni' streamlining (the bullet above), so the order is right.